modules/rosa-getting-support.adoc

```diff
@@ -18,7 +18,7 @@ If you experience difficulty with a procedure described in this documentation, v
 .. Complete the remaining fields.
 .. On the _Review_ page, select the correct cluster ID that you are contacting support about, and click `Submit`.
 
-You can also get support from link:https://aws.amazon.com/premiumsupport/[AWS Support] as long as you have a valid AWS Support contract and have enabled AWS Business, Enterprise On-Ramp, or Enterprise support plans.
+An AWS Business or Enterprise Support plan is not required to obtain support from AWS for ROSA, but we recommend that ROSA customers enable at least AWS Business Support for 24x7 access to technical support and architectural guidance. For more information about AWS Support plans, see link:https://aws.amazon.com/premiumsupport/[AWS Support].
```

Comment:
The [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#w) specifies avoiding the usage of _we_ in technical documentation. The IBM Style guide also has a helpful section on [personal pronouns](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#personal-pronouns).

Reply:
Hi @michaelryanpeter, this content is originally sourced from the following AWS ROSA documentation page that has been updated to resolve customer complaints about the AWS support language for ROSA. 

https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshooting-rosa.html#rosa-support

The ROSA Support Implementation Plan specifies the need to maintain information parity on AWS support language between AWS and Red Hat documentation sources. @smunje1 and I have been collaborating on these updates with the goal of maintaining information consistency. The above text has been editing and approved by our AWS editorial staff for publication on docs.aws.amazon.com/rosa. If the stylistic choices in the above text are a cause for concern, perhaps we could take the alternative approach of linking out to this information from https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshooting-rosa.html#rosa-support, rather than natively hosting it in the OpenShift docs. @smunje1 @michaelryanpeter what are your thoughts on this approach?

Reply:
@codymant Thank you for the additional context! To be honest, I feel a little out of my depth here, but I do think linking to the support statement would be my preferred approach for style consistency.

I'm looping in @AndrewJones-RH and @kalexand-rh.

Reply:
I too think adding a link is the best solution for this context. Here is what I am planning to do:

"An AWS Business or Enterprise Support plan is not required to obtain support from AWS for ROSA. For more information, see [AWS Support](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshooting-rosa.html#rosa-support). 

Please let me know what you think?

Reply:
I too think adding a link is the best solution for this context. Here is what I am planning to do:

"An AWS Business or Enterprise Support plan is not required to obtain support from AWS for ROSA. For more information, see [AWS Support](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshooting-rosa.html#rosa-support)."

Please let me know what you think?

Reply:
@smunje1 I agree with this approach. I would also suggest adding the following text in bold: 

"An AWS Business or Enterprise Support plan is not required to obtain support from AWS for ROSA, **but AWS does recommend that ROSA customers enable at least AWS Business Support**. For more information, see [AWS Support](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshooting-rosa.html#rosa-support)."

Reply:
@codymant The above approach is fine with me.

@michaelryanpeter Would you like to share your thoughts? If not, I'd like to add these updates to the PR. Thank you!

Cc @kalexand-rh @AndrewJones-RH

Reply:
I'd change "AWS does recommend" to "AWS recommends" to simplify the verb tense, but I agree that adding a link to the AWS support docs in this way is the best approach.

Reply:
@kalexand-rh Agreed on the suggested revision. Thank you for the feedback.

Reply:
Thank you @codymant and @kalexand-rh. I have incorporated your suggestion.